### ISSUE TITLE
Please make it possible to import the lexer classes from the outside

### ISSUE THREAD

**Issue by jgonggrijp** [i18next/i18next-parser#532]
## 🚀 Feature Proposal

Currently the `exports` field in the `package.json` forbids importing anything from the package other than `dist/index.js`. I would like to request that either (1) the `exports` field is extended so that the classes in `dist/lexers/*.js` can be imported as well, or (2) those classes are re-exported from `dist/index.js` in the same way as is currently the case for the default exports from `dist/{broccoli,parser,transform}.js`. This would enable third parties to extend those lexers.

I would be happy to submit a pull request that implements either option.

## Motivation

I want to implement a custom lexer that can parse the syntax supported by [handlebars-i18next](https://github.com/UUDigitalHumanitieslab/handlebars-i18next). Such a lexer could reuse about half of the code that is already in `dist/lexers/{base,handlebars}-lexer.js`, so I would like to subclass `HandlebarsLexer`. This is currently impossible. I can work around it by copy-pasting the code, but a proper import and extend is much more sustainable.

## Example

```js
import BaseHbsLexer from 'i18next-parser/dist/lexers/handlebars-lexer.js';

class CustomHbsLexer extends BaseHbsLexer {
    constructor(options = {}) {
        options.functions = options.functions || ['i18n'];
        super(options);
    }

   // more customizations
}
```

**Comment by karellm** [comment]
This should be fixed in `6.0.1`. Please note that this version dropped support for Node 12 also.

**Comment by jgonggrijp** [comment]
Thanks!
